adagrams/game.py

```diff
@@ -1,11 +1,158 @@
+import random
+
+
 def draw_letters():
-    pass
+    """
+    Builds and returns a list of 10 random letters
+    for the user to pick from based on contents in letter_bank 
+    """
+
+    letter_bank = {
+        "A": 9,
+        "B": 2,
+        "C": 2,
+        "D": 4,
+        "E": 12,
+        "F": 2,
+        "G": 3,
+        "H": 2,
+        "I": 9,
+        "J": 1,
+        "K": 1,
+        "L": 4,
+        "M": 2,
+        "N": 6,
+        "O": 8,
+        "P": 2,
+        "Q": 1,
+        "R": 6,
+        "S": 4,
+        "T": 6,
+        "U": 4,
+        "V": 2,
+        "W": 2,
+        "X": 1,
+        "Y": 2,
+        "Z": 1,
+    }
+
+    # Initialize variables
+    letter_bank_keys = letter_bank.keys()
+    letter_bank_copy = letter_bank.copy()
+    letters = []
+    hand_size = 0
+
+    # Hand draw logic, including incrementer and non-zero value check
+    while hand_size < 10:
+        letter = random.choice(list(letter_bank_keys))
```

Comment:
```suggestion
        letter = random.choice(list(letter_bank))
```

The default iterator over a dictionary is its keys so the `keys()` method can be removed and we'll experience the same result.